### PR TITLE
TVAULT-5133 : Add provision to restrict/allow RHOSP16 containers upload

### PR DESCRIPTION
**Pull request by rajneeshkapoor**

### CODE REVIEW

File: redhat-director-scripts/docker/build_rhosp16_containers.sh [hunk 1]
```diff
@@ -55,22 +56,39 @@ do
...
       cp Dockerfile_${openstack_release} Dockerfile
       buildah bud --format docker -t docker.io/trilio/trilio-datamover:${tvault_version}-${openstack_release} .
-      podman push --authfile /root/auth.json docker.io/trilio/trilio-datamover:${tvault_version}-${openstack_release}
-
+      if [ $(echo "$PUSH_PKG_CNT" | tr '[:upper:]' '[:lower:]') == "yes" ]
+      then
+          echo "$PUSH_PKG_CNT : Push Datamover Containers to Docker")
+          podman push --authfile /root/auth.json docker.io/trilio/trilio-datamover:${tvault_version}-${openstack_release}
+      else
+          echo "$PUSH_PKG_CNT : Don't push Datamover Containers to Docker")
```
Comment (savitha-p) [inline redhat-director-scripts/docker/build_rhosp16_containers.sh:64]: Remove `)`

File: redhat-director-scripts/docker/build_rhosp16_containers.sh [hunk 2]
```diff
@@ -9,13 +9,14 @@ echo -e "\tYou can use following commands:"
...
-   echo "Script takes exacyly 1 argument"
-   echo -e "./build_container.sh <tvault_version>"
+if [ $# -lt 2 ];then
+   echo "Script takes exactly 2 arguments"
+   echo -e "./build_container.sh <tvault_version> <PUSH_PKG_CNT>"
    exit 1
 fi
 
 tvault_version=$1
+PUSH_PKG_CNT=$2
```
Comment (shyam-biradar) [inline redhat-director-scripts/docker/build_rhosp16_containers.sh:19]: @rajneeshkapoor For what purpose we introduced this variable?

Comment (rajneeshkapoor) [reply redhat-director-scripts/docker/build_rhosp16_containers.sh:19]: It's a provision at Jenkins job level to restrict/allow RHOSP16 (_& others as well; for now kolla_) containers upload to respective registry. If this variable set to no or anything other than "yes", code flow will create containers (_& packages too_), but will not upload them, thereby preventing accidental overwriting.

File: redhat-director-scripts/docker/build_rhosp16_containers.sh (hunk 1 above)
Comment (rajneeshkapoor) [reply redhat-director-scripts/docker/build_rhosp16_containers.sh]: Resolved

**Review by savitha-p — APPROVED** [review]

File: redhat-director-scripts/docker/build_rhosp16_containers.sh (hunk 2 above)
Comment (shyam-biradar) [reply redhat-director-scripts/docker/build_rhosp16_containers.sh:19]: Okay.

**Review by shyam-biradar — APPROVED** [review]

**Pull request merged**